.github/workflows/ci.yml

```
@@ -228,7 +228,7 @@ jobs:
       - uses: actions/checkout@v5
       - uses: ./.github/actions/setup-rust
         with:
-          toolchain: nightly
+          toolchain: nightly-2025-06-26
       - name: Rust Lint - Format
         run: cargo +nightly fmt --all --check
```

@joseph-isaacs this tries to use the latest nightly (2025-08-25) which fails. Maybe we need to pin all the +nightly annotations?

```suggestion
        run: cargo +nightly-2025-06-26 fmt --all --check
```